SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LegacyLoadBalancingStrategy.cpp

```diff
@@ -15,6 +16,8 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialLegacyLoadBalancing)
 
+//#define BENCH_INTEREST_PERF
```

Comment:
Remove?

Reply:
Gonna keep it in to make it easier to follow Nic's workings